---
pdg:
    morseHuman : false
    niutHuman : false
    groupHuman : false
    mocapHuman : false
    adreamMocapHuman : true
    adaptationHuman : false
    fullHumanConfig : true
